.changeset/khaki-hounds-think.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-github-actions': minor
+---
+
+Github Workflow Runs UI is modified to show in Card view instead of table, with branch selection option
```

Comment:
This should be updated to say it's an optional Card view, some may still want to use the Table view

Reply:
> This should be updated to say it's an optional Card view, some may still want to use the Table view

Agree, its optional card view